tests/unit/via/views/test___init__.py

```diff
@@ -4,13 +4,14 @@
 
 
 class TestIncludeMe:
-    config = mock.MagicMock()
+    def test_include_me(self):
```

Comment:
Putting the tests inside a test method, so that `pytest` is aware these are tests, rather than just failing at the top level on the asserts.